spec/services/moves/import_people_spec.rb

```diff
@@ -27,16 +27,16 @@
      { prison_number: prisoner_two.nomis_prison_number, first_name: 'Bob' }]
   end
   let(:personal_care_needs_response) do
-    [{ offender_no: prisoner_one.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU9' },
-     { offender_no: prisoner_two.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU9' },
-     { offender_no: prisoner_two.nomis_prison_number, problem_type: 'MATSTAT', problem_code: 'ACCU4' }]
+    [{ offender_no: prisoner_one.nomis_prison_number, problem_type: 'FOO', problem_code: 'AA' },
```

Comment:
These keys plumb together the correct fallback. Previously the key returned for the problem code happened to be pregnancy by accident.

Reply:
it's not clear from the test why this is returning `FOO` as problem_type,
does it means that it is a problem type we are not handling?

Reply:
maybe you want just to add a context to make it clear:

context "when the response has problem_type which ...." do

end
or anything that can clarify that `problem_type: 'FOO'`

Reply:
Yeah, I use foo because internally this value isn't used to control the behaviour of anything

Reply:
I can probably just remove the problem type attribute